pikaday.js

```diff
@@ -1249,9 +1344,32 @@
                 if (typeof this._o.onOpen === 'function') {
                     this._o.onOpen.call(this);
                 }
+
+                if (this._o.field && this._o.field != this._o.trigger) {
+                  this.speak(this.getLabel());
+                }
             }
         },
 
+        getLabel: function () {
+    			var label = '',
+    				opts = this._o;
+    			if (opts.field && opts.field.id) {
+    				label = document.querySelector('label[for="' + opts.field.id + '"]');
+    				label = label ? label.textContent || label.innerText : '';
+    			}
+    			if (!label && opts.trigger) {
+    				label = opts.trigger.textContent || opts.trigger.innerText;
+    			}
+    			label += ' (' + opts.i18n.help + ')';
```

Comment:
Currently, i checked this will return `(undefined)`. I think the reason is that we don't have a `id` for the input. We can add an `id` [here](https://github.com/Kaligo/kaligo-web/pull/7401/files#diff-ab40f7a6fe2ec5d7d1f43a9479fe804ec171c5666dda883c831a55ac274f81a1R20) right